### DISCUSSION TITLE
Update release.zip of sync_template repo

### DISCUSSION THREAD

**Discussion by h7b** [ObsidianPublisher/obsidian-github-publisher#166] [Q&A — Template] [answered]
Hi Mara,

I'm using the current [update.yml](https://github.com/ObsidianPublisher/actions/blob/main/template/update.yml) from your [actions repo](https://github.com/ObsidianPublisher/actions) as guided from [the docs](https://obsidian-publisher.netlify.app/changelog/github%20action%20&%20reusable%20workflows/).

However, the script always asks to add the deprecated workflows (which are `auto_update.yml`, `create_index.yml` and `update_req_clean_medias.yml`). The reason might be that the latest version `release.zip` is [2.2.0](https://github.com/ObsidianPublisher/sync_template/releases/tag/2.2.0), which also contains the old actions.

It will be better to remove the actions from the [sync_template repo](https://github.com/ObsidianPublisher/sync_template)? Since you already introduced the new `actions repo`.

**Comment by Mara-Li** [comment, marked as answer]
Hello!
Release was done :)
